### PR TITLE
Subproperty sets

### PR DESCRIPTION
**Pull request by AronNovak**

### CODE REVIEW

File: plugins/restful/RestfulEntityBase.php [hunk 1]
```diff
@@ -698,7 +698,18 @@ protected function setPropertyValues(EntityMetadataWrapper $wrapper, $null_missi
         $field_value = $this->propertyValuesPreprocess($property_name, $request[$public_field_name], $public_field_name);
       }
 
-      $wrapper->{$property_name}->set($field_value);
```
Comment (ordavidil) [inline plugins/restful/RestfulEntityBase.php:701]: @AronNovak 
can you please add an elaborated comment about this change?
thanks!

**Review by ordavidil — CHANGES_REQUESTED** [review]

File: plugins/restful/RestfulEntityBase.php (hunk 1 above)
Comment (AronNovak) [reply plugins/restful/RestfulEntityBase.php:701]: @ordavidil added a few comments

Comment (ordavidil) [reply plugins/restful/RestfulEntityBase.php:701]: cool, can you also push it?   :)
thanks !

Comment (AronNovak) [reply plugins/restful/RestfulEntityBase.php:701]: :) Now i did

Comment (ordavidil) [reply plugins/restful/RestfulEntityBase.php:701]: thanks!

**Review by ordavidil — APPROVED** [review]

**Pull request merged**